pkg/cmd/config.go

```diff
@@ -90,6 +96,20 @@ func setConfigValue(key, value string) error {
 	return saveConfig()
 }
 
+func listAllConfig(out io.Writer) error {
+	config, err := unmarshalConfig()
+	if err != nil {
+		return nil
+	}
+
+	// Iterating over structs is hard and there are only two configuration
+	// directives at time of writing. Hard-code the output format for now, and
+	// invest in a more maintainable method in the future.
+	_, err = fmt.Fprintf(out, "user.name=%s\nuser.email=%s\n",
```

Comment:
I considered this, and wanted the `Config` object to be the source of all the configuration, as it is the known list of valid configuration directives.

For example, if someone manually edited the `config.yml` they could make unknown configuration directives appear in this way. Worse, they could add nested keys, which would result in confusing rendering without special handling.